src/configuration/services/mysql.md

```diff
@@ -11,11 +11,15 @@ The service types `mariadb` and `mysql` both refer to MariaDB for compatibility
 * mariadb:10.0
 * mariadb:10.1
 * mariadb:10.2
+* mariadb:10.3
+* mariadb:10.4
 
 
 * mysql:10.0
 * mysql:10.1
 * mysql:10.2
+* mysql:10.3
+* mysql:10.4
```

Comment:
Let's not include the `mysql` aliases for new ones.  People shouldn't be using them.  We want to transition to using just the accurate labels.